src/sentry/api/bases/sentryapps.py

```diff
@@ -45,7 +45,15 @@ def wrapped(self, *args, **kwargs):
 
 class SentryAppsPermission(SentryPermission):
     scope_map = {
-        'GET': (),  # Public endpoint.
+        # Public endpoint.
+        'GET': ('event:read',
+                'event:write',
+                'event:admin',
+                'project:releases',
+                'project:read',
+                'org:read',
```

Comment:
@MeredithAnya What about the `org:write` permission? Do all users who have `org:write` also have `org:read`?

Reply:
yeah: https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1231